### DISCUSSION TITLE
pybricks and blockly

### DISCUSSION THREAD

**Discussion by Altlogik** [pybricks/support#1177] [General]
Good evening

It's probably a fantasy

For younger children, I would really like block programming.
It would be great to combine pybricks and blockly

It would be possible to give functions tied to blocks, and it would be easier for the guys to program.

Users could prescribe these functions themselves.

**Comment by NotFenixio** [comment]
_Edited_: This would be really cool actually! There are some features available in Pybricks that are not present in apps like SPIKE.

**Reply by laurensvalk** [reply]
Except for the Technic Hub, SPIKE Essential Hub, Inventor Hub, City Hub, and Move Hub :smile:

**Reply by NotFenixio** [reply]
Oh, sorry, I don't know why my mind read SPIKE. I'll edit my comment.

**Comment by laurensvalk** [comment]
Great call @Altlogik. You might be surprised what we've been working on for the past 6 months. Stay tuned! :tada:
